app/controllers/DraftMetadataChecksResultsController.scala

```diff
@@ -92,23 +95,34 @@ class DraftMetadataChecksResultsController @Inject() (
       s"Require details message for $key"
   }
 
+  private def getAffectedProperties(errorReport: Option[ErrorFileData]): Set[String] = {
+    errorReport match {
+      case Some(er) if er.fileError == FileError.SCHEMA_REQUIRED => errorReport.get.validationErrors.flatMap(_.errors.map(_.property)).toSet
```

Comment:
We have matched for the content of this option already, so can do `er.validationErrors` directly rather than `errorReport.get.validationErrors`